app/controllers/editor/categories_controller.rb

```diff
@@ -43,6 +43,24 @@ def destroy
     redirect_to editor_categories_url, notice: "カテゴリが削除されました"
   end
 
+  # move to left
+  def move_higher
```

Comment:
Can you some actions except CRUD as other controller?
ref: http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/